README.md

```diff
@@ -46,6 +46,10 @@ hvml compile <file>
 ```
 This will output the compiled file to stdout.
 
+## Machine Integer
```

Comment:
It is a bit strange to have this section right after how to use and right before syntax. It's worded as a limitation of the language and generally we don't want to advertise in bold letters how limited we are.

I'd say this would be better placed  2 paragraphs below, when you're defining the syntax of a number.
"Here we define a function `two` as returning the number 2:
...
Currently, the only supported type of machine numbers are unsigned 60-bit integers.
..."

Also, it would be nice to have a paragraph explaining binary and hexa number syntax and mention the possibility of having digit separators (123_456 vs 0b101011_11010 vs 0x123_abc).